Changelog.md

```diff
@@ -5,6 +5,7 @@ Language Features:
 
 Compiler Features:
  * Metadata: Added support for IPFS hashes of large files that need to be split in multiple chunks.
+ * Yul: Calldata array slicing is available when compiling via Yul
```

Comment:
Since [Sol->Yul] (as you have noticed by now ;-)), is still rather incomplete and experimental, so far we haven't added Changelog entries for it. We might change that policy soon, but for now, I'd keep it at no changelog entry for these.

Reply:
Ok, understandable :)